### PR TITLE
keyboard_locks: merge formats

### PR DESCRIPTION
**Pull request by lasers**
Merging all `icon_*` config parameters into the `format` config parameter.

### CODE REVIEW

File: py3status/modules/keyboard_locks.py [hunk 1]
```diff
@@ -1,86 +1,69 @@
...
 
 Configuration parameters:
     cache_timeout: refresh interval for this module (default 1)
-    format: display format for this module (default '{caps} {num} {scr}')
-    icon_caps_off: show when Capitals Lock is off (default 'CAPS')
-    icon_caps_on: show when Capitals Lock is on (default 'CAPS')
-    icon_num_off: show when Numeric Lock is off (default 'NUM')
-    icon_num_on: show when Numeric Lock is on (default 'NUM')
-    icon_scr_off: show when Scroll Lock is off (default 'SCR')
-    icon_scr_on: show when Scroll Lock is on (default 'SCR')
```
Comment (lasers) [inline py3status/modules/keyboard_locks.py]: Deprecating things is not my strong suit. Check if the params here exists in the user's config.

File: py3status/modules/keyboard_locks.py [hunk 2]
```diff
@@ -1,36 +1,39 @@
...
+}
+```
+
 SAMPLE OUTPUT
-[
-    {'color': '#00FF00', 'full_text': 'CAPS '},
-    {'color': '#00FF00', 'full_text': 'NUM '},
-    {'color': '#FF0000', 'full_text': 'SCR'},
-]
+{'color': '#00FF00', 'full_text': 'CAPS NUM SCR'},
```
Comment (tobes) [inline py3status/modules/keyboard_locks.py]: I think it is good to have sample output showing the different colors like we had.

File: py3status/modules/keyboard_locks.py (hunk 1 above)
Comment (tobes) [reply py3status/modules/keyboard_locks.py]: > Deprecating things is not my strong suit.

Then you need to be very careful about adding things you may later want to remove.

I like the idea of this PR (simplifying things) I'm going to think about how we can make this sort of thing easier to do for users ie nicer formats.

**Comment by lasers** [comment]
Sensible solution is to hide locks by default. I don't think we use `scroll_lock` anywhere anymore so it may be a good idea to keep it out of default too.

**Comment by ultrabug** [comment]
Good enough, default is backward compatible

**Comment by lasers** [comment]
Sweet. I have a different branch that allows users to click on locks instead of using their keyboard. I was wondering about this problem of consolidating everything into one `format`. We misses out on indexes... That's one of many questions I want to know. py3status is so cool!

**Pull request merged**